Zilon.Core/Zilon.BotMassLauncher/Program.cs

```diff
@@ -16,11 +16,17 @@ class Program
         private static bool _isInfinite;
         private static ulong _infiniteCounter;
         private static string _botMode;
+        private static string _scorePath;
+        private static string _botCatalog;
+        private static string _botAssembly;
+        private static string _schemeCatlogPath;
         private static CancellationToken _shutdownToken;
         private static CancellationTokenSource _shutdownTokenSource;
 
         static void Main(string[] args)
         {
+            Console.WriteLine("[x] START");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310821)